src/main/native/cpp/view/AnalyzerPlot.cpp

```diff
@@ -127,6 +132,17 @@ void AnalyzerPlot::SetData(const Storage& data, const std::vector<double>& ff,
                                          slow[i].velocity);
     m_data[kChartTitles[3]].emplace_back(slow[i].timestamp - t,
                                          slow[i].acceleration);
+
+    if (i > 0) {
+      // If the current timestamp is in the startTimes array, it is the
+      // beginning of a new test and the dt will be inflated. Therefore we skip
+      // it to exclude that dt and effectively reset dt calculations.
+      if (std::find(startTimes.begin(), startTimes.end(), slow[i].timestamp) ==
+          startTimes.end()) {
+        double dt = (slow[i].timestamp - slow[i - 1].timestamp) * 1000;
```

Comment:
If `i` is the start time, subtracting the timestamp at `i - 1` will result in a large dt.

Reply:
Doesn't the if statement skip over values where the timestamp at `i` is a start time?